ci_build/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Image for launching TensorFlow Haskell tests.
+FROM tensorflow/tensorflow:nightly-devel
+# gcr.io/tensorflow/tensorflow:latest-devel
+MAINTAINER TensorFlow authors <tensorflow-haskell@googlegroups.com>
+
+RUN \
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 575159689BEFB442 && \
+    echo 'deb http://download.fpcomplete.com/ubuntu trusty main'| tee /etc/apt/sources.list.d/fpco.list
+
+RUN apt-get update
```

Comment:
To optimize docker's caching, I think it would be best to merge these commands under a single "RUN" command.